plugins/auth-backend/package.json

```diff
@@ -80,6 +81,7 @@
     "yn": "^4.0.0"
   },
   "devDependencies": {
+    "@backstage/backend-defaults": "workspace:^",
```

Comment:
Not used I think? 🤔

Reply:
It is now, I merged #19455 into this one